packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -484,8 +483,8 @@ export default defineComponent({
           })
         }
       }
-
       this.createModal(modal)
+      this.setModalInputErrorMessage(this.$gettext("Password can't be empty"))
```

Comment:
This is because the validation on the input only starts when you at least pressed a key once. Therefore there is no initial message warning that empty passwords are not valid. The message will be gone once at least one character is typed in.

Reply:
Do we really need this ? Is it consistent with the other models ?

Reply:
Either way: an instant error message is not good UX IMO. The user has not typed anything, why show an error?

Reply:
removed, fine with me